.circleci/config.yml

```diff
@@ -952,7 +952,7 @@ jobs:
       - TEST_RESULTS: /tmp/test-results
       - CONSUL_IMAGE: "docker.mirror.hashicorp.services/hashicorppreview/consul-enterprise:1.13-dev"
       - ENVOY_IMAGE: "envoyproxy/envoy:v1.22.2"
```

Comment:
Nothing critical but I think envoy should be v1.22.4 (this is what our values file has as there was bug the forced us to go to that version)